pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -357,21 +350,26 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			return err
 		}
 
-		if reg.Prefix == "" {
-			if reg.Location == "" {
-				return &InvalidRegistries{s: "invalid condition: both location and prefix are unset"}
-			}
-			reg.Prefix = reg.Location
-		} else {
+		// Allow config authors to always use Prefix.
```

Comment:
I don't find the comment helpful. I'd prefer a comment stating that either prefix OR location OR both must be set.